charts/opensearch/templates/statefulset.yaml

```diff
@@ -232,6 +232,17 @@ spec:
           - name: "{{ template "opensearch.uname" . }}"
             mountPath: {{ .Values.opensearchHome }}/data
 {{- end }}
+{{- if .Values.sysctlInitContainer }}
```

Comment:
For consistency could we change this to work the same way as the persistence container?
- `sysctlInit.enabled` to enable the functionality
- `sysctlInit.image` and `sysctlInit.imageTag` to control those field.
- default to `busybox:latest` since we don't need any opensearch scripts.